### ISSUE TITLE
Chris

### ISSUE THREAD

**Issue by SubtleHyperbole** [cshanes/vue_js_dashboard#1]
Hey Chris, sorry to place this in such a random place, but I couldn’t find another way to get a message to you. 

I read your blog post about working on using machine learning models to predict DFS performance in baseball.  I am presently working on a similar project right now, and would love if i could bounce some ideas off you.  Perhaps we could even work on it together—if you are still interested in trying to accomplish that.  

Based on your write up and my own experience working on this the last month and a half, I think the key is finding the correct approach.  For one, I think instead of using regression to predict specific scores, it is easier to instead have the target be score ranges — like 0, 0.1-5, 5-10, 10-15, etc.. this way you would be able to use non-regression algorithms with Categorical targets.  Not only that, but these would be ordinal categories. I have had some success so far, but I could really use someone to bounce ideas off of.

Anyways, I would love to discuss it more with you.  My email is subtle .... hyperbole.  () master at gmail. You can figure out what that address should be. 

Basically all that minus the spaces and punctuation.(I’m avoiding the email scrapers).  Hope to hear from you. -Paul
